kortex_driver/src/hardware_interface.cpp

```diff
@@ -1040,6 +1006,125 @@ void KortexMultiInterfaceHardware::sendTwistCommand()
   base_.SendTwistCommand(k_api_twist_command_);
 }
 
+void KortexMultiInterfaceHardware::resetFaults()
+{
+  if (in_fault_ == 0.0)
+  {
+    reset_fault_async_success_ = 1.0;
+    reset_fault_cmd_ = NO_CMD;  // clear the reset_fault_cmd
+    RCLCPP_INFO(LOGGER, "Arm is not in a faulted state. Returning SUCCESS from resetFaults().");
+    return;
+  }
+  try
+  {
+    // change servoing mode first
+    servoing_mode_hw_.set_servoing_mode(k_api::Base::ServoingMode::SINGLE_LEVEL_SERVOING);
+    base_.SetServoingMode(servoing_mode_hw_);
+    // wait for the switch to apply
+    std::this_thread::sleep_for(std::chrono::milliseconds(500));
```

Comment:
Please create a constant variable defined at the top of the file for the reset wait time.